### ISSUE TITLE
Sailboat sails cannot be controlled while in "Immersive First person Mode" in Survival game mode.

### ISSUE THREAD

**Issue by ProfessorDragonTheOriginal** [anegostudios/VintageStory-Issues#7456]
The cleat on a Sailboat that controls the sails (up/down) cannot be reached by the player model while in "Immersive First person Mode", while in a "Survival" game mode. Therefore the sails cannot be controlled.

<img width="248" height="601" alt="Image" src="https://github.com/user-attachments/assets/fa543b76-9f41-46d6-bc92-0dcf4a00f5cd" />

Originally reported on the VS Forum here:
https://www.vintagestory.at/forums/topic/18176-i-sit-in-the-sailboat-and-i-cannot-access-the-sails-unless-i-swim-over-and-unfurl-them/#comment-91340

**Steps to Reproduce Issue**
1. Enable the Experimental mode Settings, Interface, "Immersive First person Mode" (sic).

<img width="553" height="53" alt="Image" src="https://github.com/user-attachments/assets/b84073ff-7c0e-411b-8b23-d001820eb848" />

2. Be in Survival game mode: /gamemode Survival
3. Be in the pilot (starboard rear) seat of a sailboat.
4. The Halyard Cleat can not longer be accessed.

Access is restored by (A) turning off this mode or (B) by switching to Creative mode, which has a longer reach value.

MINOR TYPO
The "Immersive First person Mode" has an unusual case formatting.
I recommend "Immersive First Person Mode".
I recommend having a review of case formatting in the Settings in general (may need to be another issue) as it is not consistent.
* Green - sentence case or single words
* Orange - lower case
* Red - weird mix
* Purple - I don't know. Maybe okay? Check your Style Manual.
* 
<img width="932" height="705" alt="Image" src="https://github.com/user-attachments/assets/403cee43-1b89-494b-a498-f36d9e584f21" />

Thanks, Professor Dragon.

**Comment by Heegrim** [comment]
Yeah this is so annoying, pls just make the cleat hitbox a tiny bit bigger in 1.22. 

A workaround is placing a seat right in front of the pilot’s seat and use that to reach the sails. If you look all the way to the right you can get back in the pilot seat.
